pkg/api/graph/interfaces.go

```diff
@@ -38,7 +38,7 @@ const (
 type Markers []Marker
 
 // MarkerScanner is a function for analyzing a graph and finding interesting things in it
-type MarkerScanner func(g Graph, f Namer) []Marker
+type MarkerScanner func(g Graph, f Namer, chaseRefs bool) []Marker
```

Comment:
@deads2k I don't like this but do you have any better idea for how to pass info down to the marker level that --chase-refs is being used?